### PR TITLE
feat(components): [select-v2]add collapse-tags-tooltip

### PR DESCRIPTION
**Pull request by Alanscut**
same function in select from pr #6245 
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by jw-foss — COMMENTED** [review]
The code LGTM, still left a comment for you to consider 🏓

File: packages/components/select-v2/src/select.vue [hunk 1]
```diff
@@ -66,13 +66,50 @@
...
-                    :style="{
-                      maxWidth: `${tagMaxWidth}px`,
-                    }"
-                    >+ {{ modelValue.length - 1 }}</span
+                  <el-tooltip
+                    :disabled="dropdownMenuVisible || !collapseTagsTooltip"
+                    :fallback-placements="['bottom', 'top', 'right', 'left']"
+                    :effect="effect"
+                    placement="bottom"
+                    :teleported="false"
```
[the remark this answers is not in the document]
Comment (Alanscut) [reply packages/components/select-v2/src/select.vue:74]: you mean change 
`:disabled="dropdownMenuVisible || !collapseTagsTooltip"` 
to 
`
:disabled="dropdwonMenuVisible
v-if="collapseTagsTooltip"
`?

[the remark this answers is not in the document]
Comment (JeremyWuuuuu) [reply packages/components/select-v2/src/select.vue:74]:
```suggestion
                  <el-tooltip
                    v-if="collapseTagsTooltip"
                    :disabled="dropdownMenuVisible"
                    :fallback-placements="['bottom', 'top', 'right', 'left']"
                    :effect="effect"
                    placement="bottom"
                    :teleported="false"
```

[the remark this answers is not in the document]
Comment (Alanscut) [reply packages/components/select-v2/src/select.vue:74]: if use `v-if`， it should follow a `<span...>`  with `v-else` too.

[the remark this answers is not in the document]
Comment (JeremyWuuuuu) [reply packages/components/select-v2/src/select.vue:74]: We can do the same like https://github.com/element-plus/element-plus/pull/6331/files does.

[the remark this answers is not in the document]
Comment (Alanscut) [reply packages/components/select-v2/src/select.vue]: you can check the new [commit](https://github.com/element-plus/element-plus/pull/6532/commits/38fb9bde1749580acc47db994e5d948b69bc8d5a)

**Review by jw-foss — APPROVED** [review]

File: packages/components/select-v2/src/select.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (JeremyWuuuuu) [reply packages/components/select-v2/src/select.vue]: LGTM! Thanks

**Comment by Alanscut** [comment]
ok,i will add the same change(v-if v-else) on #6245

**Pull request merged**